### DISCUSSION TITLE
Error after 0.6.0 update

### DISCUSSION THREAD

**Discussion by Thailanda** [ekutner/home-connect-hass#188] [Community Support]
Hello guys,

after updating the integration to version 0.6.0 I get following configuration error:

Integration error: home_connect_alt - Requirements for home_connect_alt not found: ['home-connect-async==0.7.5'].

I'm new to home-assistant, sorry if this might be a stupid question. Could anyone point me in the right direction?

Thanks in advance.

**Comment by Stubennatter** [comment]
I had the same problem, what is to do? Roll back to old Version not show the option to do this
